Resources/Locale/en-US/components/storage-component.ftl

```diff
@@ -1,9 +1,9 @@
 comp-storage-no-item-size = N/A
 comp-storage-cant-insert = Can't insert.
-comp-storage-insufficient-capacity = Insufficient capacity.
+comp-storage-too-big = Too big!
+comp-storage-insufficient-capacity = No room!
 comp-storage-invalid-container = This doesn't go in there!
 comp-storage-anchored-failure = Can't insert an anchored item.
 comp-storage-cant-drop = You can't let go of { THE($entity) }!
 comp-storage-window-title = Storage Item
-comp-storage-window-volume = Items: { $itemCount }, Stored: { $usedVolume }/{ $maxVolume }
-comp-storage-window-volume-unlimited = Items: { $itemCount }
+comp-storage-window-volume = Items: { $itemCount }/{ $maxCount }
```

Comment:
If the storage has a max total weight, it should also show that ratio in the UI. Ideally it'd also show the weight of each individual item, but that'd require more significant UI changes, so it's probably fine to leave as is for now